package.json

```diff
@@ -97,6 +98,7 @@
     "babel-preset-stage-0": "^6.24.1",
     "babel-runtime": "^6.23.0",
     "chai": "^3.5.0",
+    "chai-as-promised": "^5.3.0",
```

Comment:
Please, provide for all additional dependencies info why do we need it and where is it used

Reply:
Debug leftovers, fixed